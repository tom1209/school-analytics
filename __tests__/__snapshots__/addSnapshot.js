import { render } from "@testing-library/react";
import AddStudent from "../../pages/add";

it('renders add student page unchanged', () => {
  const { container } = render(<AddStudent />)
  expect(container).toMatchSnapshot()
})