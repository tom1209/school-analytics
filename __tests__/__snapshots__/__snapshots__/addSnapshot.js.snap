// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders add student page unchanged 1`] = `
<div>
  <div
    class="addContainer"
  >
    <div
      style="position: fixed; z-index: 9999; top: 16px; left: 16px; right: 16px; bottom: 16px; pointer-events: none;"
    />
    <div
      class="addStudentFormContainer"
    >
      <form
        autocomplete="off"
      >
        <h1
          class="addStudentTitle"
        >
          Add a student and grades to a school
        </h1>
        <div
          class="formField"
        >
          <label
            for="schoolName"
          >
            School Name
          </label>
          <input
            id="schoolName"
            name="schoolName"
            value=""
          />
        </div>
        <div
          class="formField"
        >
          <label
            for="studentName"
          >
            Student Name
          </label>
          <input
            id="studentName"
            name="studentName"
            value=""
          />
        </div>
        <div
          class="formField"
        >
          <h3
            styles="gradeTitle"
          >
            Add Grades
          </h3>
          <div
            class="addNewGradesContainer"
          >
            <label
              for="newClass"
            >
              Enter Class
            </label>
            <label
              for="newMark"
            >
              Enter Mark
            </label>
            <input
              id="newClass"
              name="newClass"
              value=""
            />
            <input
              id="newMark"
              name="newMark"
              value=""
            />
          </div>
          <button
            class="addNewGradeButton"
            id="addNewGradeButton"
            type="button"
          >
            Add More Grades
          </button>
        </div>
        <button
          class="addStudentButton"
          id="addStudentButton"
          type="submit"
        >
          Add Student
        </button>
      </form>
    </div>
  </div>
</div>
`;
